syntax = "proto3";

service Tasks {
  rpc AddTask (AddTaskRequest) returns (AddTaskResponse);
}

message AddTaskRequest {
  string id = 1;
  string title = 2;
  string status = 3;
  string due_date = 4;
}

message AddTaskResponse {
  string id = 1;
}